standard.md

```diff
@@ -6,17 +6,17 @@
-**8. Adherence to Standards & Best Practices** | Projects must demonstrate adherence to standards, best practices, and/or principles. For example, [the Principles for Digital Development](https://digitalprinciples.org/principles/).
-**9. Do No Harm By Design** | All projects must demonstrate that they have taken steps to ensure the project anticipates, prevents, and does no harm.
-**9a) Data Privacy & Security** | Projects collecting data must identify the types of data collected and stored. Projects must also demonstrate how they ensure the privacy and security of this data in addition to the steps taken to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9b) Inappropriate & Illegal Content** | Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials in addition to mechanisms for detecting, moderating and removing inappropriate/illegal content.
-**9c) Protection from Harassment** | If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users. 
+**1. Relevance to Sustainable Development Goals** | Digital public goods must be designed and developed to advance the [Sustainable Development Goals (SDGs)](https://sdgs.un.org/goals) and demonstrate such by providing links/documentation that support their relevance. 
+**2. Use of Approved Open Licenses** | Digital public goods must demonstrate the use of an approved open license. For open-source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content collections the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. DPGs are encourageed to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. See [The full license list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md) for reference.
+**3. Clear Ownership** | Ownership of assets that the digital public good produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
```

Comment:
In general things like copyrights, trademarks, etc., can not have a creator status as another object (a DPG) but rather is required to be an individual (human or corporate) which could be the project community as a whole, individual contributor(s), or a third party organization that might be the copyright holder for some individuals' contributions.

Reply:
Yes, we do not require the owner to be DPG, but ownership to be clearly defined. This prevents many tail risk scenarios where dependence is created on some object, and that object is owned by a commercial entity who may not distribute it as a public good.